src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -84,7 +90,11 @@
                                                                   :gradient-cover?     true
                                                                   :customization-color :purple}])
                              :emoji               "🍑"}}]
-       [quo/account-overview temp/account-overview-state]
+       [quo/account-overview
+        {:current-value       (utils/prettify-balance balance)
+         :account-name        (:name account)
+         :account             :default ;; for now
```

Comment:
ideally remove the comments, better to make a follow up issue to track this 👍

Reply:
can put it in with the issue to get the other missing details on this page

Reply:
Done